README.md

```diff
@@ -1,2 +1,41 @@
-# Edge AI GStreamer Apps
-> Repository to host GStreamer based Edge AI applications for TI devices
+# Edge AI TIOVX Apps
+> Repository to host TIOVX based Edge AI applications for TI devices
+
+
+## Steps to run
+
+1. Clone this repo in your target under /opt
+
+    ```console
+    root@j7-evx:/opt# git clone https://github.com/TexasInstruments/edgeai-tiovx-apps.git
```

Comment:
@s-jain1 Is this supposed to say "j7-evm" instead of "j7-evx"?

Reply:
Updated this